internal/types/types.go

```diff
@@ -45,6 +45,7 @@ type ConfigurationPolicyOptions struct {
 	PruneObjectBehavior    string             `json:"pruneObjectBehavior,omitempty" yaml:"pruneObjectBehavior,omitempty"`
 	RecordDiff             string             `json:"recordDiff,omitempty" yaml:"recordDiff,omitempty"`
 	RecreateOption         string             `json:"recreateOption,omitempty" yaml:"recreateOption,omitempty"`
+	CustomMessage          *CustomMessage     `json:"customMessage,omitempty" yaml:"customMessage,omitempty"`
```

Comment:
Similar to `EvaluationInterval`, I'm not sure there's a benefit to making this a pointer--unless there was a reason to do so?
```suggestion
	CustomMessage          CustomMessage     `json:"customMessage,omitempty" yaml:"customMessage,omitempty"`
```